joblib/test/test_numpy_pickle.py

```diff
@@ -278,9 +359,9 @@ def test_compressed_pickle_dump_and_load():
     # or smaller than cache_size)
     for cache_size in [0, 1e9]:
         try:
-            dumped_filenames = numpy_pickle.dump(
-                expected_list, fname, compress=1,
-                cache_size=cache_size)
+            dumped_filenames = numpy_pickle.dump(expected_list,
+                                                 fname, compress=1,
```

Comment:
Agreed, this change was just related to personal preferences (my editor is not in cause ;) ).